src/main/java/org/squiddev/cobalt/lib/TableLib.java

```diff
@@ -114,7 +114,7 @@ protected Varargs invoke(LuaState state, DebugFrame di, Varargs args) throws Lua
 				case 0: { // "sort" (table [, comp]) -> void
 					LuaTable table = args.arg(1).checkTable();
 					LuaValue compare = args.isNoneOrNil(2) ? NIL : args.arg(2).checkFunction();
-					int n = table.prepSort();
+					int n = table.length();
```

Comment:
Can you remove the `prepSort` function from `LuaTable? I assume it's not used anywhere now?

Reply:
Yes it's now unused, done.